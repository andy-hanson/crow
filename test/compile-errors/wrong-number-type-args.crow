magic opt ?out(a opt ?in)
	todo

test opt ?out(a opt ?in)
	a magic<?out>
