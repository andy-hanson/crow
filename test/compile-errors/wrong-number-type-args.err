/wrong-number-type-args.nz 5:7-5:18 there are functions named 'magic', but none takes 1 type arguments. candidates:
opt<?out>(opt<?in>) (from wrong-number-type-args.nz line 1)
