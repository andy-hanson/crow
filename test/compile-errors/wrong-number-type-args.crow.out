exit code: 0
stderr:
test/compile-errors/wrong-number-type-args.crow 8:7-8:12 There are functions named 'magic', but none takes 1 type arguments. candidates:
	'magic out?(a in?)' (from test/compile-errors/wrong-number-type-args.crow line 4)
